src/Network/Network/Network.csproj

```diff
@@ -14,6 +14,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.PowerShell.AutoMapper" Version="6.2.2" />
+    <PackageReference Include="Microsoft.Azure.Management.Network" Version="26.0.0" />
```

Comment:
You will probably need to update ALL the csproj file that require this update? Might want to confirm with PS team though.

Reply:
These are instructions by Mikhail to enable network sdk. Only this is needed.

Reply:
You can see many other files that need to be updated from this PR:
![image](https://user-images.githubusercontent.com/49326906/232586600-ce1cc8ca-a3c8-4e4b-b349-c3d0233377a4.png)

Probably PS team might ask you to do so.

Reply:
Actally I had those files in the PR, then the PS team asked me to remove them, will check again tonight